### PR TITLE
ADEN-12160 Adding Media.Net display configuration

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: src/ad-bidders/prebid/adapters/medianet.ts [hunk 1]
```diff
@@ -12,7 +13,15 @@ export class Medianet extends PrebidAdapter {
 		return Medianet.bidderName;
 	}
 
-	prepareConfigForAdUnit(code, { cid, crid }: PrebidAdSlotConfig): PrebidAdUnit {
+	prepareConfigForAdUnit(code, { sizes, cid, crid }: PrebidAdSlotConfig): PrebidAdUnit {
+		if (context.get(`slots.${code}.isVideo`)) {
```
Comment (nandy-andy) [inline src/ad-bidders/prebid/adapters/medianet.ts:17]: We could unit test it! 🤞

File: src/platforms/shared/bidders/bidders-state.setup.ts [hunk 2]
```diff
@@ -22,39 +26,25 @@ export class BiddersStateSetup implements DiProcess {
...
-			context.set('bidders.prebid.rubicon_pg.enabled', this.instantConfig.get('icPrebidRubiconPG'));
-			context.set('bidders.prebid.rubicon.enabled', this.instantConfig.get('icPrebidRubicon'));
-			context.set('bidders.prebid.telaria.enabled', this.instantConfig.get('icPrebidTelaria'));
-			context.set(
-				'bidders.prebid.triplelift.enabled',
-				this.instantConfig.get('icPrebidTriplelift'),
-			);
-			context.set('bidders.prebid.verizon.enabled', this.instantConfig.get('icPrebidVerizon'));
+
+			this.enableBidder('appnexus', 'icPrebidAppNexus');
```
Comment (nandy-andy) [inline src/platforms/shared/bidders/bidders-state.setup.ts:30]: If you want to improve readability you could iterate through a dictionary and call `this.enableBidder()` within the loop 🤔

File: src/ad-bidders/prebid/adapters/medianet.ts [hunk 3]
```diff
@@ -47,4 +56,24 @@ export class Medianet extends PrebidAdapter {
 			],
 		};
 	}
+
+	private getStandardConfig(code, sizes, cid, crid): PrebidAdUnit {
```
[the remark this answers is not in the document]
Comment (nandy-andy) [reply src/ad-bidders/prebid/adapters/medianet.ts:60]: Oh, and I see the official docs call it probably a "standard implementation": https://support.google.com/admanager/answer/12273163?hl=en

[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/ad-bidders/prebid/adapters/medianet.ts:60]: Yeah, `display` is not really a common industry name, and `standard` here means a normal implementation in which bids can be rendered using `pbjs.renderAd` - no VASTs, no Safe Frame messages etc :)

File: src/platforms/shared/bidders/bidders-state.setup.ts [hunk 4]
```diff
@@ -71,4 +61,12 @@ export class BiddersStateSetup implements DiProcess {
 			context.get('bidders.prebid.enabled') || context.get('bidders.a9.enabled'),
 		);
 	}
+
+	private enableBidder(name: string, icKey: string): void {
```
[the remark this answers is not in the document]
Comment (dmnsobczak) [reply src/platforms/shared/bidders/bidders-state.setup.ts:65]: How about `enableBidderOrNot`? xD But agree, renamed 👍

File: src/platforms/shared/bidders/bidders-state.setup.ts (hunk 2 above)
Comment (dmnsobczak) [reply src/platforms/shared/bidders/bidders-state.setup.ts:30]: Good suggestion - changed 👍

File: src/ad-bidders/prebid/adapters/medianet.ts (hunk 1 above)
Comment (dmnsobczak) [reply src/ad-bidders/prebid/adapters/medianet.ts:17]: Done!

File: spec/ad-bidders/prebid/adapters/medianet.spec.ts [hunk 5]
```diff
@@ -0,0 +1,101 @@
+import { Medianet } from '@wikia/ad-bidders/prebid/adapters/medianet';
+import { expect } from 'chai';
+import { context } from '@wikia/core';
+
+describe('Medianet bidder adapter', () => {
```
Comment (nandy-andy) [inline spec/ad-bidders/prebid/adapters/medianet.spec.ts:5]: ❤️

**Review by nandy-andy — APPROVED** [review]

**Pull request merged**